kframework/article.md

```diff
@@ -1,12 +1,16 @@
-In this blog post we will be exploring, describing, and dissecting the first phase of the collaboration between Runtime Verification and Serokell! The collaboration involved the work on making optimizations to [K](https://kframework.org/), a rewrite-based executable semantic framework in which programming languages, type systems and formal analysis tools can be defined with the help of configurations and rewrite rules. The blog post will contain the details and the results of the discovery phase, the recommendations which are introduced in the Findings section, and the implementation of 2 global tasks, alongside their performance results.
+In this blog post, we will be exploring, describing, and dissecting the first phase of the collaboration between Runtime Verification and Serokell. 
```

Comment:
There are now lots of instances of one trailing space being present at the end of some of the lines. Are they supposed to be there? If not, can I remove them?

Reply:
No, you can remove them. :)